test/run_tests_and_server.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
With the proper node utilities, it should be possible to get this part to work even on Windows.

Reply:
I have done some testing in windows and attempted to rewrite the testing script as a .bat but I cannot verify it works because  #162 - I couldn't get webgazerExtracClient.py to work with windows.